<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Portable management of Hotel </string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../Pictures/8DC3.gif</normaloff>../../../../Pictures/8DC3.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="RecordButton" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>320</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Record And Check out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshButton" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>320</y>
      <width>102</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>160</y>
      <width>102</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_Edit" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>120</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>The Residue Room Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Input Name or Num</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool" >
    <property name="title" >
     <string>Tool</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionConfig" />
    <addaction name="actionInitial_The_Database" />
    <addaction name="actionAdd_Room_Info" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>about</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuTool" />
   <addaction name="menuAbout" />
  </widget>
  <action name="actionConfig" >
   <property name="text" >
    <string>Config</string>
   </property>
  </action>
  <action name="actionDf" >
   <property name="text" >
    <string>df</string>
   </property>
  </action>
  <action name="actionAccount" >
   <property name="text" >
    <string>Account</string>
   </property>
  </action>
  <action name="actionInitial_The_Database" >
   <property name="text" >
    <string>Initial Database</string>
   </property>
  </action>
  <action name="actionAdd_Room_Info" >
   <property name="text" >
    <string>Add Room Info</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
